rviz_common/src/rviz_common/properties/property_tree_widget.cpp

```diff
@@ -119,9 +119,13 @@ void PropertyTreeWidget::setModel(PropertyTreeModel * model)
 void PropertyTreeWidget::propertyHiddenChanged(const Property * property)
 {
   if (model_) {
-    setRowHidden(
-      property->rowNumberInParent(), model_->parentIndex(property),
-      property->getHidden() );
+    const auto& parent_index = model_->parentIndex(property);
```

Comment:
nit: just a moment ago you submitted a PR where the reference symbol `&` had a space between it and the preceding keyword, which seemed consistent with the rest of the file. Should this not be:

```suggestion
    const auto & parent_index = model_->parentIndex(property);
```

If we have linting enabled in CI it might pick this up, no?